bin/control_rods

```diff
@@ -148,3 +165,11 @@ set_cookie(Node) ->
             erlang:set_cookie(Node, list_to_atom(Cookie))
     end.
 
+measure(CtrlRod, Value) ->
+    Node = node_name(),
+    Metric = metric(CtrlRod),
+    rpc:call(Node, folsom_metrics, notify, [Metric, Value, gauge]).
+
+metric(CtrlRod) ->
+    list_to_binary([<<"logplex.control_rod.">>, CtrlRod, <<".active">>]).
```

Comment:
`logplex.control-rod` ?  

ref: https://github.com/heroku/engineering-docs/blob/master/guides/metric-naming-conventions.md

Reply:
Thanks for pointing this out. `shh` does the normalization [here](https://github.com/heroku/shh/blob/master/mm.go#L54-L60) for all metrics provided by folsom.